sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/experimental/Cancelable.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.maps.plugin.viewport.experimental
+
+/**
+ * Allows to cancel the associated asynchronous operation.
+ */
+fun interface Cancelable {
```

Comment:
Did you check whether there's already a Cancelable defined somewhere else? I thought we used this pattern with high-level animations.

Reply:
💯 with @macdrevx - we do use them in camera plugin

Reply:
yes, we have, but it's inside the animations package, not sure if we want to reuse that one directly. One solution would be to use the animation plugin's Cancelable inside viewport for now, and in v11, refactor the Cancelable to a generic package for reuse. cc @kiryldz

Reply:
@pengdev `Cancelable` interface is already defined in `sdk-base` and not in animation plugin itself so why not re-use it? especially taken into account viewport plugin depends on camera animation one

Reply:
I meant the package name 😄 
```
package com.mapbox.maps.plugin.animation
```
This class is very generic, we should move it to generic package in next major release.

For now I will just reuse the Cancelable from animation plugin.

Reply:
Created an internal ticket to track the work in the next major release.